src/library/basecoverartdelegate.cpp

```diff
@@ -140,4 +140,9 @@ void BaseCoverArtDelegate::paintItem(
             return;
         }
     }
+    // As a fallback use the (background) color a placeholder
+    // until the cover has been loaded
+    if (coverInfo.color) {
```

Comment:
In what circumstances does this evaluate to false? Can we make this a `VERIFY_OR_DEBUG_ASSERT`?

Reply:
Thanks for improving the comment, but I wondered why this is wrapped in an `if` statement. Is color not always set? Now that I though about this, I realized that this probably for cases when the track either has no image at all or when is shown for the first time, so the cover art has never been opened yet and hence does not have a color. So we can't make this a debug assertion, right?

Reply:
By using an optional member in CoverInfo we clearly state that the color might not always be available. All dependent code has to account for this. If we need to document it then in CoverInfo, but not here.